Leaking global definition of importScripts

**Bug**

When you use the loader with other packages downstream, and those packages try to assert a web worker environment, the global declaration of `importScripts` conflicts with that, causing errors like:

```
error TS2451: Cannot redeclare block-scoped variable 'importScripts'.
```

Since it is only doing this for internal purposes and it doesn't _own_ that declaration, and that declaration is a block scoped variable of a `const`, this makes some consumers 😢.

Ideally the loader would use `"lib": [ "webworker" ]` or would just declare the variable not in the global type scope.

Package Version: 0.2.0

--- a/src/interfaces.d.ts
+++ b/src/interfaces.d.ts
@@ -236,4 +236,4 @@ declare interface NodeRequire {
 	undef(moduleId: string, recursive?: boolean): void;
 }
 
-declare const arguments: IArguments;
+declare var arguments: IArguments;

--- a/src/loader.ts
+++ b/src/loader.ts
@@ -3,9 +3,8 @@ import ModuleShim = DojoLoader.ModuleShim;
 import Module = DojoLoader.Module;
 import Package = DojoLoader.Package;
 
-declare const load: (module: string) => any;
-declare const Packages: {} | undefined;
-declare const importScripts: ((url: string) => void);
+// Nashorn global
+declare var Packages: { [key: string]: any; } | undefined;
 
 (function (args?: string[]): void {
 	let globalObject: any = (function (): any {
@@ -23,6 +22,12 @@ declare const importScripts: ((url: string) => void);
 		}
 		return {};
 	})();
+
+	// Nashorn global
+	const load: (module: string) => any = globalObject.load;
+
+	// webworker global
+	const importScripts: ((url: string) => void) = globalObject.importScripts;
 
 	const EXECUTING = 'executing';
 	const ABORT_EXECUTION: Object = {};

